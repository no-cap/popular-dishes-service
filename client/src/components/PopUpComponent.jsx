import React from 'react';
import PhotoBox from './PhotoBox.jsx';
import ReviewsBox from './ReviewsBox.jsx';
import { 
  SmallDescriptionFormatter,
  SmallDescriptionBorder, 
  ItemDescription, 
  ReviewsFormatter,
  Name,
  Price
} from '../components-style/PopupComponent-style.jsx';

class PopUpComponent extends React.Component {
  constructor(props) {
    super(props)
    this.photoBoxElement = React.createRef();
    this.resetPhotoBox = this.resetPhotoBox.bind(this);
    this.handleKeyPress = this.props.handleKeyPress;
  }

  resetPhotoBox() {
    this.photoBoxElement.current.resetCurrent();
  }

  componentDidMount() {
    document.addEventListener("keydown", this.props.handleKeyPress);
  }

  componentWillUnmount() {
    document.removeEventListener("keydown", this.props.handleKeyPress, false);
  }

  render() {
    return (
      <ItemDescription>
        <PhotoBox photos={this.props.photos} ref={this.photoBoxElement} />
        <SmallDescriptionFormatter>
            <Name>{this.props.item.dish_name}</Name>
            <Price>${this.props.item.price}.00</Price>
        </SmallDescriptionFormatter>
        <ReviewsFormatter>
          <SmallDescriptionBorder> <p>{this.props.item.description}</p> </SmallDescriptionBorder>
          <ReviewsBox reviews={this.props.reviews} />
        </ReviewsFormatter>
      </ItemDescription>

    )
  }
}

export default PopUpComponent;
