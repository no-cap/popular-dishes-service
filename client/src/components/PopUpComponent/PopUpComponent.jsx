import React from 'react';
import PhotoBox from '../PhotoBox/PhotoBox';
import ReviewsBox from '../ReviewsBox/ReviewsBox';
import {
  SmallDescriptionFormatter,
  SmallDescriptionBorder,
  ItemDescription,
  ReviewsFormatter,
  Name,
  Price
} from './PopupComponentStyles.jsx';

class PopUpComponent extends React.Component {
  constructor(props) {
    super(props)
    this.photoBoxElement = React.createRef();
    this.resetPhotoBox = this.resetPhotoBox.bind(this);
    this.handleKeyPress = this.props.handleKeyPress;
  }

  componentDidMount() {
    document.addEventListener('keydown', this.props.handleKeyPress);
  }

  componentWillUnmount() {
    document.removeEventListener('keydown', this.props.handleKeyPress, false);
  }

  resetPhotoBox() {
    this.photoBoxElement.current.resetCurrent();
  }

  render() {
    const { photos, dishName, price, description, reviews } = this.props.dish;
    const { host } = this.props;
    return (
      <ItemDescription>
        <PhotoBox photos={photos} ref={this.photoBoxElement} />
        <SmallDescriptionFormatter>
          <Name>{dishName}</Name>
          <Price>{`${price}`}</Price>
        </SmallDescriptionFormatter>
        <ReviewsFormatter>
          <SmallDescriptionBorder>
            <p>{description}</p>
          </SmallDescriptionBorder>
          <ReviewsBox reviews={reviews} host={host} />
        </ReviewsFormatter>
      </ItemDescription>
    );
  }
}

export default PopUpComponent;
