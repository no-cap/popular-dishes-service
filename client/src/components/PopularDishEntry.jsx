import React from 'react';
import $ from 'jquery';

import { 
  PhotoImage, 
  Cropper, 
  PopularDish, 
  PopularDishText, 
  PopularDishName, 
  PriceBox,
  Price
} from '../components-style/PopularDishEntry-style.jsx';

var notFound = 'https://www.yorkshirecareequipment.com/wp-content/uploads/2018/09/no-image-available.jpg';



class PopularDishEntry extends React.Component {
  constructor(props) {
    super(props);
    this.state = {

    };
    this.onClickHandler = this.onClickHandler.bind(this);
    this.handleBlackSpace = this.handleBlackSpace.bind(this)
  }

  onClickHandler(e) {
    e.preventDefault();
    this.props.buttonHandler(e, this.props.item, this.props.photos);
  }
  // attempt at handling different sized images. A lot harder than anticipated;
  handleBlackSpace(url) {
    var temporaryImage = new Image()
    temporaryImage.src = url;
    $(temporaryImage).one('load', function () {
      var orgWidth = temporaryImage.width;
      var orgHeight = temporaryImage.height;
      if ((orgWidth / orgHeight) < (192 / 140)) {
        return 'tooShort';
      }
      if ((orgHeight / orgWidth) > (140 / 192)) {
        return 'tooTall';
      }
    });
  }

  render() {
    return (
      <div onClick={this.onClickHandler}>
        <PopularDish className="popper">
          <Cropper>
            {(this.props.photos.length > 0) ? 
              <PhotoImage src={this.props.photos[0].url} size={this.handleBlackSpace(this.props.photos[0].url)} className="popper"/> : 
              <PhotoImage src={notFound} size={this.handleBlackSpace(notFound)} className="popper"/>
            }
          </Cropper>
          <PopularDishText className="popper">
            <PopularDishName className="popper">{this.props.item.dish_name}</PopularDishName>
            <br/>
            <span className="popper"> {this.props.photos.length} Photos &#183; {this.props.item.review_count} Reviews </span>
          </PopularDishText>
          <PriceBox className="popper">
            <Price primary className="popper">${this.props.item.price}</Price>
            <Price className="popper">${this.props.item.price}</Price>
          </PriceBox>
        </PopularDish>
      </div>
    )
  }
}

export default PopularDishEntry;
