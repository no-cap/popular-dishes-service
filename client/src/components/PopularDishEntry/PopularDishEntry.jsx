import React from 'react';
import $ from 'jquery';

import {
  PhotoImage,
  Cropper,
  PopularDish,
  PopularDishText,
  PopularDishName,
  PriceBox,
  Price,
} from './PopularDishEntryStyles';

const notFound = 'https://www.yorkshirecareequipment.com/wp-content/uploads/2018/09/no-image-available.jpg';


class PopularDishEntry extends React.Component {
  constructor(props) {
    super(props);
    this.onClickHandler = this.onClickHandler.bind(this);
    this.handleBlackSpace = this.handleBlackSpace.bind(this);
  }

  onClickHandler(e) {
    e.preventDefault();
    this.props.buttonHandler(e, this.props.dish, this.props.photos);
  }

  // attempt at handling different sized images. A lot harder than anticipated;
  // eslint-disable-next-line class-methods-use-this
  handleBlackSpace(url) {
    const temporaryImage = new Image();
    temporaryImage.src = url;
    $(temporaryImage).one('load', () => {
      const orgWidth = temporaryImage.width;
      const orgHeight = temporaryImage.height;
      if ((orgWidth / orgHeight) < (192 / 140)) {
        return 'tooShort';
      }
      if ((orgHeight / orgWidth) > (140 / 192)) {
        return 'tooTall';
      }
    });
  }

  render() {
    const { photos, dish } = this.props;
    return (
      <div onClick={this.onClickHandler} >
        <PopularDish className="popper">
          <Cropper>
            {(photos.length > 0)
              ? <PhotoImage src={photos[0].url} size={this.handleBlackSpace(photos[0].url)} className="popper" />
              : <PhotoImage src={notFound} size={this.handleBlackSpace(notFound)} className="popper" />}
          </Cropper>
          <PopularDishText className="popper">
            <PopularDishName className="popper">{dish.dishName}</PopularDishName>
            <br />
            <span className="popper">{photos.length} Photos &#183; {dish.reviews.length} Reviews</span>
          </PopularDishText>
          <PriceBox className="popper">
            <Price primary className="popper">${dish.price}</Price>
            <Price className="popper">${dish.price}</Price>
          </PriceBox>
        </PopularDish>
      </div>
    );
  }
}

export default PopularDishEntry;
