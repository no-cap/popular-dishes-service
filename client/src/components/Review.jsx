import React from 'react';
import StarBox from './starbox.jsx';
import Star from './star.jsx';
import PersonIcon from './personIcon.jsx';
import StarType from '../components-style/TypesOfStars.jsx';

import { 
    SingleReview, 
    PhotoCropper, 
    ProfilePic, 
    UserName, 
    TinyPairIcon, 
    FriendCount, 
    TinyStar, 
    UserReviews,
    StarContainer,
    Date,
    ReviewText
} from '../components-style/SingleReview.jsx'

class Review extends React.Component {
    constructor(props) {
        super(props);
        this.state = {
            text: this.props.data.text,
            readMore: false
        };
        this.onClickHandler = this.onClickHandler.bind(this);
        this.dateFormatter = this.dateFormatter.bind(this);
    }
    // handles clicking on Read more / Read less
    onClickHandler(e) {
        this.setState({ readMore: !this.state.readMore })
    }
    // Formats Dates
    dateFormatter(date) {
        var arr = date.slice(0, 11).split("-");
        if (arr[1][0] === '0') {
            var dateWithZero = arr[1] + '/' + arr[0] + '/' + arr[2];
            return dateWithZero.slice(1);
        } else {
            return arr[1] + '/' + arr[0] + '/' + arr[2];
        }
    }

    render() {
        return (
            <SingleReview >
                <PhotoCropper> <ProfilePic src={this.props.data.userphoto} /> </PhotoCropper>
                <UserName> {this.props.data.username} </UserName>
                <TinyPairIcon > 
                    <PersonIcon /> 
                </TinyPairIcon>
                <FriendCount > {this.props.data.friends} </FriendCount>
                <TinyStar> 
                    <StarType type="mini" rating={4}> <Star/> 
                </StarType> </TinyStar>
                <UserReviews> {this.props.data.reviews} </UserReviews>
                <StarContainer> 
                    <StarBox rating={this.props.data.rating} /> 
                </StarContainer>
                <Date> {this.dateFormatter(this.props.data.date)} </Date>
                <ReviewText>
                    <div id={this.props.data.user}>{(this.state.readMore) ? this.state.text : this.state.text.split(" ").slice(0, 22).join(" ")}</div>
                    {(this.props.data.text.split(" ").length > 22) ? <button onClick={this.onClickHandler}> {(this.state.readMore) ? 'Read less' : 'Read more'} </button> : null}
                </ReviewText>
            </SingleReview>
        )
    }
}

export default Review;


// { JSON.stringify(this.props.data) }