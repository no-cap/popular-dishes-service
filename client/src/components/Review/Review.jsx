import React from 'react';
import StarBox from '../Starbox/Starbox';
import Star from '../Starbox/Star';
import PersonIcon from '../PersonIcon/PersonIcon';
import StarType from '../Starbox/StarStyles';

import {
  SingleReview,
  PhotoCropper,
  ProfilePic,
  UserName,
  TinyPairIcon,
  FriendCount,
  TinyStar,
  UserReviews,
  StarContainer,
  Date,
  ReviewText,
} from './ReviewStyles';

class Review extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
      text: this.props.data.text,
      readMore: false,
    };
    this.onClickHandler = this.onClickHandler.bind(this);
    this.dateFormatter = this.dateFormatter.bind(this);
  }

  // handles clicking on Read more / Read less
  onClickHandler(e) {
    this.setState({ readMore: !this.state.readMore });
  }

  // Formats Dates
  // eslint-disable-next-line class-methods-use-this
  dateFormatter(date) {
    const arr = date.slice(0, 11).split('-');
    if (arr[1][0] === '0') {
      const dateWithZero = `${arr[1]}/${arr[0]}/${arr[2]}`;
      return dateWithZero.slice(1);
    }
    return `${arr[1]}/${arr[0]}/${arr[2]}`;
  }

  render() {
    return (
      <SingleReview>
        <PhotoCropper>
          <ProfilePic src={this.props.data.userphoto} />
        </PhotoCropper>
        <UserName>{this.props.data.username}</UserName>
        <TinyPairIcon>
          <PersonIcon />
        </TinyPairIcon>
        <FriendCount>{this.props.data.friends}</FriendCount>
        <TinyStar>
          <StarType type="mini" rating={4}><Star /></StarType>
        </TinyStar>
        <UserReviews>{this.props.data.reviews}</UserReviews>
        <StarContainer>
          <StarBox rating={this.props.data.rating} />
        </StarContainer>
        <Date>{this.dateFormatter(this.props.data.date)}</Date>
        <ReviewText>
          <div id={this.props.data.user}>{(this.state.readMore) ? this.state.text : this.state.text.split(' ').slice(0, 22).join(' ')}</div>
          {(this.props.data.text.split(' ').length > 22) ? (
            <button onClick={this.onClickHandler}>{(this.state.readMore) ? 'Read less' : 'Read more'}</button>
          ) : null}
        </ReviewText>
      </SingleReview>
    );
  }
}

export default Review;
