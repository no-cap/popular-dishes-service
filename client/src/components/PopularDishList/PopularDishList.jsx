import React from 'react';
import $ from 'jquery';
import PopularDishEntry from '../PopularDishEntry/PopularDishEntry';
import Modal from '../Modal/Modal';
import PopUpComponent from '../PopUpComponent/PopUpComponent';
import { Slider, Slide, GoLeft, GoRight, NextDish, LastDish, DishArrowRight, DishArrowLeft } from './PopularDishListStyles';
import { CloseButton, ModalStyle, CloseFormat, DishButtons } from '../Modal/ModalStyles';

const notFound = 'https://www.yorkshirecareequipment.com/wp-content/uploads/2018/09/no-image-available.jpg';

class PopularDishList extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
      modalVisibility: false,
      scrollPosition: 5,
      dish: {},
      photos: [],
      index: 0,
    };
    this.onClickHandler = this.onClickHandler.bind(this);
    this.onChangeHandler = this.onChangeHandler.bind(this);
    this.setView = this.setView.bind(this);
    this.popUpComponentElement = React.createRef();
    this.outsideModalHandler = this.outsideModalHandler.bind(this);
    this.handleKeyPress = this.handleKeyPress.bind(this);
  }

  // eslint-disable-next-line class-methods-use-this
  onClickHandler(e) {
    e.preventDefault();
    if (e.target.id === 'goRight') {
      $('.slider').animate({ scrollLeft: '+=630' }, 100);
    } else {
      $('.slider').animate({ scrollLeft: '-=630' }, 100);
    }
  }

  onChangeHandler(e) {
    e.preventDefault();
    this.setState({ scrollPosition: $('.slider').scrollLeft() });
  }

  setView(e, dish, photos) {
    const { popularDishes } = this.props;
    const { index, modalVisibility } = this.state;

    if (e.preventDefault) {
      e.preventDefault();
    }
    if (e.target.className === 'closeIt' || e.target.id === 'closeModal' || e.target.className.includes('popper')) {
      this.setState({ dish: dish || {} }); // setting new dish
      this.setState({ photos: photos || [] }); // setting new photos
      this.setState({ modalVisibility: !modalVisibility }); // turning modal visibility to TRUE
      // setting index of current dish in popular dishes
      popularDishes.forEach((popularDish, i) => ((JSON.stringify(popularDish.dishName) === JSON.stringify(dish)) ? this.setState({ index: i }) : null));
    } else if (e.target.id === 'nextDish') {
      this.popUpComponentElement.current.resetPhotoBox();
      if (popularDishes[index + 1].photos.length > 0) {
        this.setState({ dish: popularDishes[index + 1] }); // setting next dish
        this.setState({ photos: popularDishes[index + 1].photos }); // setting next photos
        this.setState({ index: this.state.index + 1 }); // setting index to one more
      } else {
        this.setState({ dish: popularDishes[index + 1] }); // setting next dish
        this.setState({ photos: [{ url: notFound }] }); // setting next photos
        this.setState({ index: index + 1 }); // setting index to one more
      }
    } else if (e.target.id === 'previousDish') {
      this.popUpComponentElement.current.resetPhotoBox();
      if (popularDishes[index - 1].photos.length > 0) {
        this.setState({ dish: popularDishes[index - 1] }); // setting to previous dish
        this.setState({ photos: popularDishes[index - 1].photos }); // setting previous photos
        this.setState({ index: index - 1 }); // setting index to one less
      } else {
        this.setState({ dish: popularDishes[index - 1] }); // setting to previous dish
        this.setState({ photos: [{ url: notFound }] }); // setting previous photos
        this.setState({ index: index - 1 }); // setting index to one less
      }
    }
  }

  outsideModalHandler(e) {
    if (e.target.className.includes('modal')) {
      this.setState({ modalVisibility: false });
    }
  }

  handleKeyPress(e) {
    const escapeObj = { target: { value: '', className: 'closeIt', id: 'closeModal' } };
    if (e.keyCode === 27) this.setView(escapeObj);
  }

  render() {
    const { scrollPosition, modalVisibility, index, dish: dish, photos } = this.state;
    const { popularDishes, host } = this.props;

    return (
      <div>
        <Slider className="slider" onScroll={this.onChangeHandler}>
          {popularDishes.map((dish) => (
            <Slide id={dish.dishName}>
              <PopularDishEntry dish={dish} photos={dish.photos} buttonHandler={this.setView} />
            </Slide>
          ))}
        </Slider>
        {(scrollPosition > 20) ? <GoLeft onClick={this.onClickHandler} /> : null}
        {(scrollPosition < 890) ? <GoRight id="goRight" onClick={this.onClickHandler}> </GoRight> : null}
        {(modalVisibility)
          ? (
            <Modal>
              <ModalStyle className="modal" onClick={this.outsideModalHandler}>
                <CloseButton onClick={this.setView}>
                  <CloseFormat id="closeModal">Close</CloseFormat>
                  <div className="closeIt">&#x2715;</div>
                </CloseButton>
                <PopUpComponent dish={dish} ref={this.popUpComponentElement} handleKeyPress={this.handleKeyPress} host={host} />
                <DishButtons>
                  {(index < popularDishes.length - 1) ? (
                    <NextDish id="nextDish" onClick={this.setView}>
                      {popularDishes[index + 1].dishName}
                      <DishArrowRight />
                    </NextDish>
                  ) : null}
                  {(index > 0) ? (
                    <LastDish id="previousDish" onClick={this.setView}>
                      {popularDishes[index - 1].dishName}
                      <DishArrowLeft />
                    </LastDish>
                  ) : null}
                </DishButtons>
              </ModalStyle>
            </Modal>
          )
          : null}
      </div>
    );
  }
}

export default PopularDishList;
