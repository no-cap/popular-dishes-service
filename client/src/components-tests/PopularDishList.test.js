import React from 'react';
import { shallow, mount } from 'enzyme';
import PopUpComponent from '../components/PopUpComponent.jsx';
import Review from '../components/Review.jsx';
import PhotoEntry from '../components/PhotoEntry.jsx';
import PopularDishEntry from '../components/PopularDishEntry.jsx';
import PopularDishList from '../components/PopularDishList.jsx';

// for making a mock event
const mockEventGenerator = (str) => {
    return {
        target: {
            name: 'todo',
            value: str,
        },
        preventDefault: () => { },
    };
};

var sampleItem = {
    name: 'Corn Cheese',
    price: 8.50,
    description: 'Corn with Cheese'
}

var sampleReviews = [
    {
        user: 'Bob',
        text: 'I love corn cheese'
    },
    {
        user: 'Bob',
        text: 'I love corn cheese'
    },
    {
        user: 'Bob',
        text: 'I love corn cheese'
    }
];

var samplePhotos = [
    {
        url: 'https://s3-media0.fl.yelpcdn.com/bphoto/EhwQv7W3nmJqdXhvCW7pJg/o.jpg',
        caption: 'Red Dragon roll was so good!! We devoured it in record timing!'
    },
    {
        url: 'https://s3-media0.fl.yelpcdn.com/bphoto/EhwQv7W3nmJqdXhvCW7pJg/o.jpg',
        caption: 'Red Dragon roll was so good!! We devoured it in record timing!'
    },
    {
        url: 'https://s3-media0.fl.yelpcdn.com/bphoto/EhwQv7W3nmJqdXhvCW7pJg/o.jpg',
        caption: 'Red Dragon roll was so good!! We devoured it in record timing!'
    },
    {
        url: 'https://s3-media0.fl.yelpcdn.com/bphoto/EhwQv7W3nmJqdXhvCW7pJg/o.jpg',
        caption: 'Red Dragon roll was so good!! We devoured it in record timing!'
    }
];

var samplePopularDishes = [
    {
        item: sampleItem,
        photos: samplePhotos,
        reviews: sampleReviews
    },
    {
        item: sampleItem,
        photos: samplePhotos,
        reviews: sampleReviews
    },
    {
        item: sampleItem,
        photos: samplePhotos,
        reviews: sampleReviews
    },
    {
        item: sampleItem,
        photos: samplePhotos,
        reviews: sampleReviews
    },
    {
        item: sampleItem,
        photos: samplePhotos,
        reviews: sampleReviews
    },
    {
        item: sampleItem,
        photos: samplePhotos,
        reviews: sampleReviews
    },
    {
        item: sampleItem,
        photos: samplePhotos,
        reviews: sampleReviews
    },
    {
        item: sampleItem,
        photos: samplePhotos,
        reviews: sampleReviews
    },
    {
        item: sampleItem,
        photos: samplePhotos,
        reviews: sampleReviews
    },
    {
        item: sampleItem,
        photos: samplePhotos,
        reviews: sampleReviews
    }
]

describe('PopularDishList Unit test', () => {
    // just seeing if the component even renders correctly
    describe('PopularDishList should render correctly', () => {
        test('PopularDishList should render correctly in "debug" mode', () => {

            const wrapper = shallow(<PopularDishList popularDishes={samplePopularDishes} />);

            expect(wrapper).toMatchSnapshot();
        });

        test('PopularDishList should have not have the pop up initially', () => {
            const wrapper = shallow(<PopularDishList popularDishes={samplePopularDishes} />);
            expect(wrapper.find(PopUpComponent)).toHaveLength(0);
            expect(wrapper.find(PopularDishEntry)).toHaveLength(3);
            expect(wrapper.find('button')).toHaveLength(1);
        });


    });

    describe('PopularDishList Unit Interaction test', () => {
        describe('onClickHandler', () => {
            test('it should invoke the correct method when a change event is emitted', () => {
                const mockClickHandler = jest.fn();
                const wrapper = shallow(<PopularDishList popularDishes={samplePopularDishes} />);
                wrapper.instance().onClickHandler = mockClickHandler;
                wrapper.instance().forceUpdate();
                wrapper.find('button').at(0).simulate('click');
                expect(mockClickHandler).toHaveBeenCalled();
            });
            // testing that click
            test('it should correctly update the state when a change event is emitted', () => {
                const wrapper = mount(<PopularDishList popularDishes={samplePopularDishes} />);
                wrapper.find('button').simulate('click', mockEventGenerator('right'));
                wrapper.find('button').simulate('click', mockEventGenerator('right'));
                wrapper.find('button').simulate('click', mockEventGenerator('right'));
                wrapper.find('button').simulate('click', mockEventGenerator('right'));
                wrapper.find('button').simulate('click', mockEventGenerator('left'));
                wrapper.find('button').simulate('click', mockEventGenerator('left'));
                expect(wrapper.instance().state.currentView).toBe(0)
            });
        });
    });

});
