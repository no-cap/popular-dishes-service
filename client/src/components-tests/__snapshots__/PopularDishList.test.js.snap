// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopularDishList Unit test PopularDishList should render correctly PopularDishList should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
